src/modules/flow/netctl/Kconfig

```diff
@@ -0,0 +1,6 @@
+config FLOW_NODE_TYPE_NETCTL
+	tristate "Node type: netctl"
+	depends on NETCTL
+	default m
+	help
+		Network control is used to control the system network status by connman deamon
```

Comment:
You should make clear ConnMan is just one implementation for Linux. For example:

```
Network control is used to control the system network status. On Linux it will communicate with ConnMan via DBus.
```